Gopkg.toml

```diff
@@ -155,10 +155,6 @@ required = [
   name = "github.com/kr/pretty"
   revision = "cfb55aafdaf3ec08f0db22699ab822c50091b1c4"
 
-[[constraint]]
-  name = "github.com/lyft/protoc-gen-validate"
-  revision = "d75a15c1da0f853ca31399f30c49088ca5716b04"
-
```

Comment:
Go files in `vendor/github.com/cilium/proxy/go` depend on this, but we seem to install this explicitly in the Vagrantfile:
```
Vagrantfile:sudo -E /usr/local/go/bin/go get -d github.com/lyft/protoc-gen-validate
Vagrantfile:(cd ~/go/src/github.com/lyft/protoc-gen-validate ; sudo git checkout 930a67cf7ba41b9d9436ad7a1be70a5d5ff6e1fc ; make build)
```
(We also do this in the packer-ci-build)

Not sure but maybe better remove the dependency from the Vagrantfile (& packer-ci-build) and keep it as a vendor dependency? I'll need to take care that the install happens when cilium/proxy is built, but cilium/cilium does not need the binary, just the go packages.

Reply:
@jrajahalme If `vendor/github.com/cilium/proxy/go` needs the `github.com/lyft/protoc-gen-validate` then the `github.com/cilium/proxy/go` should be the one importing this.

Let me know if you need help setting up the dependencies for `github.com/cilium/proxy` because if we set up the dependencies for that project, `cilium/cilium` will inherit the constraints from `github.com/cilium/proxy`

Reply:
@aanm It seems to me the dependency is already there:
```
vendor/github.com/cilium/proxy/go/cilium/api/npds.pb.go:        _ "github.com/lyft/protoc-gen-validate/validate"
vendor/github.com/cilium/proxy/go/cilium/api/nphds.pb.go:       _ "github.com/lyft/protoc-gen-validate/validate"
vendor/github.com/cilium/proxy/go/envoy/api/v2/auth/cert.pb.go: _ "github.com/lyft/protoc-gen-validate/validate"
vendor/github.com/cilium/proxy/go/envoy/api/v2/cds.pb.go:       _ "github.com/lyft/protoc-gen-validate/validate"
...
```

Reply:
I don't know what you mean by inheriting constraints if not the above.